tests/integration/archive_test.go

```diff
@@ -83,3 +88,21 @@ func getHash(p *porter.TestPorter, path string) string {
 
 	return fmt.Sprintf("%x", h.Sum(nil))
 }
+
+func containsRequiredMetadata(p *porter.TestPorter, path string) {
+	tmpDir, err := p.FileSystem.TempDir("", "porter-integration-tests")
+	require.NoError(p.T(), err)
+	defer p.FileSystem.RemoveAll(tmpDir)
+
+	source := p.FileSystem.Abs(path)
+	l := loader.NewLoader()
+	imp := packager.NewImporter(source, tmpDir, l)
+	err = imp.Import()
+	require.NoError(p.T(), err, "opening archive failed")
+
+	_, err = p.FileSystem.Stat(filepath.Join(tmpDir, strings.TrimSuffix(filepath.Base(source), ".tgz"), "relocation-mapping.json"))
```

Comment:
I feel like this isn't giving us as much test coverage as we need here. For example, the test would pass if relocation-mapping.json is empty right? 

Can you add a check that it has the expected images in it as well?